object/user.go

```diff
@@ -83,6 +84,8 @@ type User struct {
 	AzureAD  string `xorm:"azuread varchar(100)" json:"azuread"`
 	Slack    string `xorm:"slack varchar(100)" json:"slack"`
 
+	WebAuthnCredentialList []webauthn.Credential `xorm:"blob" json:"webauthnCredential"`
```

Comment:
why list?

Reply:
Why not use string?

Reply:
this object contains []byte objects which cannot be stringfied(due to other encoding methods). If we use string here, json.Marshal will be called, which is not proper to handle[]byte. it will generate something like "[12, 13, 14]"......and dramatically increase the size it uses.

Reply:
> why list?
one user can have more than 1 credentials